vault/logical_system_paths.go

```diff
@@ -493,6 +499,77 @@ func (b *SystemBackend) rekeyPaths() []*framework.Path {
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.UpdateOperation: &framework.PathOperation{
 					Summary: "Unseal the Vault.",
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							// unseal returns `vault.SealStatusResponse` struct
+							Fields: map[string]*framework.FieldSchema{
+								"type": {
+									Type:     framework.TypeString,
```

Comment:
Do we want descriptions here? I believe it will affect code comments and intellisense when generating code